tests/integration/targets/web_application/aliases

```diff
@@ -0,0 +1,3 @@
+windows
+shippable/windows/group4
```

Comment:
I know CI is waiting on `website` to me merged but right now this repo only has 1 test group so this should be `shippable/windows/group1`